Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/LocalIncidenceCard.kt

```diff
@@ -0,0 +1,93 @@
+package de.rki.coronawarnapp.statistics.ui.homecards.cards
+
+import android.view.ViewGroup
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.HomeStatisticsCardsLocalIncidenceLayoutBinding
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass
+import de.rki.coronawarnapp.statistics.LocalIncidenceStats
+import de.rki.coronawarnapp.statistics.StatsItem
+import de.rki.coronawarnapp.statistics.ui.homecards.StatisticsCardAdapter
+import de.rki.coronawarnapp.statistics.util.formatStatisticalValue
+import de.rki.coronawarnapp.statistics.util.getContentDescriptionForTrends
+import de.rki.coronawarnapp.statistics.util.getLocalizedSpannableString
+import de.rki.coronawarnapp.ui.presencetracing.attendee.checkins.items.BaseCheckInVH.Companion.setupMenu
+import de.rki.coronawarnapp.util.StringBuilderExtension.appendWithLineBreak
+import de.rki.coronawarnapp.util.StringBuilderExtension.appendWithTrailingSpace
+import de.rki.coronawarnapp.util.formatter.getPrimaryLabel
+
+class LocalIncidenceCard(parent: ViewGroup) :
+    StatisticsCardAdapter.ItemVH<StatisticsCardItem, HomeStatisticsCardsLocalIncidenceLayoutBinding>(
+        R.layout.home_statistics_cards_basecard_layout,
+        parent
+    ) {
+
+    override val viewBinding = lazy {
+        HomeStatisticsCardsLocalIncidenceLayoutBinding.inflate(
+            layoutInflater,
+            itemView.findViewById(R.id.card_container),
+            true
+        )
+    }
+
+    override val onBindData: HomeStatisticsCardsLocalIncidenceLayoutBinding.(
+        item: StatisticsCardItem,
+        payloads: List<Any>
+    ) -> Unit = { item, _ ->
```

Comment:
I'm not sure if we need something like
`val curItem = payloads.filterIsInstance<Item>().singleOrNull() ?: item`
here, i.e. what the current update mechanic is in the nested recyclerview.
Will have to test more when everything is wired.

Reply:
Meaning if we use the diffutil to it's fullest, as to not have the UI redraw/flash unnecessary, we could get a stale `item` when scrolling, and the newer one would be in `payloads`.

Reply:
seeems to work without it during my testing but i am honestly not that informed about recycler views in this context, if you feel like this is an issue let me know and ill implement it :)

Reply:
When adding or removing local incidence cards, the "add local incidence" card flashes, so something with the stable IDs is not working correctly yet.
Once that is working we will have to use the payload mechanism mentioned above.